search/intermediate_result.cpp

```diff
@@ -112,16 +112,13 @@ RankerResult::RankerResult(double lat, double lon)
   m_region.SetParams(string(), MercatorBounds::FromLatLon(lat, lon));
 }
 
-string RankerResult::GetRegionName(storage::CountryInfoGetter const & infoGetter,
-                                   uint32_t ftype) const
+bool RankerResult::GetCountryId(storage::CountryInfoGetter const & infoGetter, uint32_t ftype,
+                                storage::TCountryId & countryId) const
```

Comment:
Is it ok that search depends on storage?

Reply:
Yes, it's OK.
```shell
y@lambda:~/coding/omim/search$ gg --max-depth 0 -h '#include "storage' | sort | uniq
#include "storage/country_decl.hpp"
#include "storage/country.hpp"
#include "storage/country_info_getter.hpp"
#include "storage/downloader_search_params.hpp"
#include "storage/index.hpp"
#include "storage/storage.hpp"
```

Reply:
Ok